systems/plants/@RigidBodyManipulator/RigidBodyManipulator.m

```diff
@@ -1078,7 +1078,12 @@
       ind = find(strcmp(jointname,lower({model.body.jointname})));
       if (robot_num~=0), ind = ind([model.body(ind).robotnum]==robot_num); end
       if (length(ind)~=1)
-        if (nargin<4 || error_level>0)
+        % if there is more than one index, check if one of them is an exact match of the search string
+        exact_matches = strcmpi({model.body.jointname},jointname);
+        ind = find(exact_matches);
+        if ( length(ind) == 1 )
+            body_ind = ind;
+        elseif (nargin<4 || error_level>0)
           error('Drake:RigidBodyManipulator:UniqueJointNotFound',['couldn''t find unique joint ' ,jointname]);
```

Comment:
thanks for another commit.  keep them coming!

but i don't actually see how this one accomplishes what you say.  The lines above already look only for exact matches (with `ind = find(strcmp(` exactly like what you're using here).  I *think* the only different is that your code does not respect the `robot_num` argument, so would check all robots (and have problems if, for instance, you have two of the same robots loaded in your model).  

btw - i definitely agree that we should have called `strcmpi` instead of calling lower on both inputs.

maybe you could commit the simplest possible test case which fails with the existing code but works with your code?